pkg/assembler/graphql/schema/certifyBad.graphql

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright 2023 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Defines a GraphQL schema for the CertifyBad. It contains the subject (which can be either a package, source or artifact),
+#  justification, origin of the attestation, and collector
+"""
+
+CertifyBad is an attestation represents when a package, source or artifact is considered bad
+
+Subject - union type that can be either a package, source or artifact object type
+Justification - string value representing why the subject is considered bad
+Origin - where this attestation was generated from (based on which document)
+Collector - the GUAC collector that collected the document that generated this attestation
+
+Note: Attestation must occur at the PackageName or the PackageVersion or at the SourceName.
+
+"""
+
+union PkgSrcArtObject = Package | Source | Artifact
+
```

Comment:
I think we should move these 3 lines below. The above `"""` comment should be attached to `CertifyBad` type